### PR TITLE
MMT-1749: add ability to submit request to generate naive variables

### PR DESCRIPTION
**Pull request by charleshuang80**
* add callout box to initiate variable generation
* add collection search controller and views for uvg
* add controller and view for submitting initial naive request
* make sure feature toggle works
* add client to talk to UVG api
* add tests

### CODE REVIEW

File: app/views/variable_generation_processes_searches/_search_results.html.erb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            <th>Short Name</th>
+            <th>Version</th>
+          </tr>
+        </thead>
+        <tbody>
+          <% if collections.any? %>
+            <% collections.each do |collection| %>
+              <tr>
+                <td class="align-c">
+                  <%= radio_button_tag('selected_collection', collection.fetch('meta', {})['concept-id'], false, class: 'selected_collection_radio_button align-c') %>
```
Comment (charleshuang80) [inline app/views/variable_generation_processes_searches/_search_results.html.erb]: look at generalizing with helper methods

File: spec/features/variable_generation/naive_variable_generation_spec.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
+describe 'Naive Variable Generation', reset_provider: true do
+  before :all do
+    @naive_uvg_col_ingest_response, @naive_uvg_col_concept_response = publish_collection_draft
+  end
+
+  before do
+    login
+
+    visit new_variable_generation_processes_search_path
```
Comment (charleshuang80) [inline spec/features/variable_generation/naive_variable_generation_spec.rb]: click the button

File: spec/features/variable_generation/naive_variable_generation_spec.rb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      within '#collection-search-results' do
+        expect(page).to have_css("input[type='radio']#selected_collection_#{@naive_uvg_col_ingest_response['concept-id']}")
+        expect(page).to have_content(@naive_uvg_col_concept_response.body['EntryTitle'])
+      end
+    end
+
+    context 'when choosing a collection and submitting to generate variables', js: true do
+      before do
+        within '#collection-search-results' do
+          puts "concept_id: #{@naive_uvg_col_ingest_response['concept-id']}"
```
Comment (charleshuang80) [inline spec/features/variable_generation/naive_variable_generation_spec.rb]: remove comments

File: app/helpers/manage_metadata_helper.rb [hunk 4]
```diff
@@ -6,7 +6,7 @@ def current_manage_title
       "manage_#{resource_type}"
     elsif controller.lookup_context.prefixes.include?('manage_cmr')
       'manage_cmr'
-    elsif controller.lookup_context.prefixes.include?('manage_variables') || controller.lookup_context.prefixes.include?('variables') || controller.lookup_context.prefixes.include?('variable_drafts') || (controller.lookup_context.prefixes.include?('collection_associations') && params[:variable_id])
+    elsif controller.lookup_context.prefixes.include?('manage_variables') || controller.lookup_context.prefixes.include?('variables') || controller.lookup_context.prefixes.include?('variable_drafts') || (controller.lookup_context.prefixes.include?('collection_associations') && params[:variable_id]) || controller.lookup_context.prefixes.include?('variable_generation_processes_searches') || controller.lookup_context.prefixes.include?('variable_generation_processes')
```
Comment (cgokey) [inline app/helpers/manage_metadata_helper.rb]: maybe consider helper method for this statement, since it is so long.

Comment (charleshuang80) [reply app/helpers/manage_metadata_helper.rb]: yeah. I think I'll look at this in the next ticket.

**Pull request merged**